packages/web-component/test-data/ej-fra/index-edit.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Insert Title Here</title>
+    <!-- Import fonts. Material Icons are needed by the web component -->
+    <link href="https://fonts.googleapis.com/css?family=Lato%7CMaterial+Icons%7CMaterial+Icons+Outlined" rel="stylesheet">
+</head>
+
+<body>
+    <!-- Here is how you declare the Web Component. Supported languages: en, fr -->
+    <read-along mode="EDIT" use-assets-folder="false" href="ej-fra.ras" audio="ej-fra.m4a" language="fra">
```

Comment:
Is `use-assets-folder="false"` required for image editing to work properly now?

Reply:
Hm, I don't think so. I haven't tried that though